nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/components/administration/workflow/dialogs/create-bucket/nf-registry-create-bucket.js

```diff
@@ -80,21 +81,22 @@ NfRegistryCreateBucket.prototype = {
      */
     cancel: function () {
         this.dialogRef.close();
-    },
-
-    /**
-     * Focus the new bucket input.
-     */
-    ngAfterViewChecked: function () {
-        this.newBucketInput.nativeElement.focus();
     }
+
+//    /**
```

Comment:
Should this section be removed? (Was it left as comment accidentally?)

Reply:
That should be removed indeed, thanks for pointing it out. I'll update the PR accordingly.